.github/workflows/ci.yml

```diff
@@ -183,9 +197,10 @@ jobs:
         with:
           config: .github/memory_statistics_config.json
           check_against: docs/doxygen/include/size_table.md
+
   proof_ci:
-    if: ${{ github.event.pull_request }}
-    runs-on: cbmc_ubuntu-latest_16-core
+    if: ${{ github.event.pull_request }} || ${{ github.event.workflow }}
+    runs-on: ubuntu-20.04
```

Comment:
These proofs run fast enough that I think it makes sense to just run them on the free runner, so people can run them on their forks without modifying the workflow file.